docs/cas-server-documentation/installation/Cassandra-ticket-registry.md

```diff
@@ -0,0 +1,118 @@
+---
+layout: default
+title: CAS - Cassandra Ticket Registry
+---
+
+# Cassandra Ticket Registry
+Cassandra integration is enabled by including the following dependency in the WAR overlay:
+
+```xml
+<dependency>
+    <groupId>org.apereo.cas</groupId>
+    <artifactId>cas-server-support-cassandra-ticket-registry</artifactId>
+    <version>${cas.version}</version>
+</dependency>
+```
+
+## Configuration Considerations
+
+There are four core configuration concerns with Cassandra:
+
+1. Keyspace
+2. Ticket cleaning
+3. Object serialization mechanism
+4. Multi datacenter replication
+
+### Keyspace
+We suggest the following Keysapace definition:
+
+- String serialized ticket:
+```cql
```

Comment:
I think it's worth taking a look. I can promise you that for the most part, most wouldn't want to setup and maintain the schema definitions, specially if the software can do itself, which makes it easier for us to  make changes later to the schema and not have to update the docs or ask deployers to run a series of scripts again. I'd encourage you to review that, either now or later as a separate step. It's something that would increase the module's adoption. Automation is key, almost always.